src/components/Accordion/AccordionTitle.tsx

```diff
@@ -47,26 +54,44 @@ class AccordionTitle extends UIComponent<ReactProps<AccordionTitleProps>, any> {
     active: PropTypes.bool,
     index: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     onClick: PropTypes.func,
+    activeIndicator: customPropTypes.itemShorthand,
+  }
+
+  static defaultProps = {
+    activeIndicator: true,
```

Comment:
My vote is for `indicator`, I'm not sure that we really need there a prefix

Reply:
I've added the prefix, so that people will know that the icon they are defining is regarding the active state, but yeah, we can make the name simpler.

Reply:
After some thoughts, agreed on the name indicator. We are nowhere using prefixes for the shorthands. Thanks for the suggestion!